src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -102,7 +102,7 @@ class AdjudicationsService(
 
         AdjudicationMappingDto(
           adjudicationNumber = nomisAdjudicationResponse.adjudicationNumber,
-          chargeSequence = nomisAdjudicationResponse.adjudicationSequence,
+          chargeSequence = nomisAdjudicationResponse.charges.first().chargeSequence,
```

Comment:
unrelated, but this code was wrong but would have always worked since both values would always be "1" - but use the correct mapping anyway to avoid confusion (there is always one and only one charge)